From ebec09d9f6c6d8ab268472347bbe7f94659d7275 Mon Sep 17 00:00:00 2001
From: Tony Luck <tony.luck@intel.com>
Date: Fri, 25 Jan 2019 15:22:34 -0800
Subject: [PATCH 25/25] numastat: Add KReclaimable to list of known fields in
 meminfo

Linux kernel added this new field in October 2018 version v4.20
with commit:

  61f94e18de94 ("mm, proc: add KReclaimable to /proc/meminfo")

Add it to numastat too.
---
 numastat.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/numastat.c b/numastat.c
index 1ae11fb..263bddf 100644
--- a/numastat.c
+++ b/numastat.c
@@ -126,7 +126,8 @@ meminfo_t system_meminfo[] = {
 	{ 31, "ShmemPmdMapped", "ShmemPmdMapped" },
 	{ 32, "HugePages_Total", "HugePages_Total" },
 	{ 33, "HugePages_Free", "HugePages_Free" },
-	{ 34, "HugePages_Surp", "HugePages_Surp" }
+	{ 34, "HugePages_Surp", "HugePages_Surp" },
+	{ 35, "KReclaimable", "KReclaimable" }
 };
 
 #define SYSTEM_MEMINFO_ROWS (sizeof(system_meminfo) / sizeof(system_meminfo[0]))
-- 
1.8.3.1

